openstack/content-repo/templates/check-deployment.yaml

```diff
@@ -54,5 +54,13 @@ spec:
             name: secret
           - mountPath: /bin-check
             name: bin
+        resources:
+          # observed usage in qa-de-1: CPU <= 0.1m, RAM = 3-4 MiB
+          limits:
+            cpu: 'l0m'
```

Comment:
what is `l0m` btw?

Reply:
10 milli, i.e. 1% of a core

Reply:
you want `10m` or `l0m` ?

Reply:
![](https://i.imgflip.com/hynx2.jpg?a451968)